src/runtime_src/core/edge/include/zynq_ioctl.h

```diff
@@ -102,7 +108,7 @@ struct drm_zocl_sync_bo {
 struct drm_zocl_info_bo {
 	uint32_t	handle;
 	uint64_t	size;
-	uint64_t	paddr;
+	addr_t	paddr;
 };
```

Comment:
1. Please keep the same indentation
2. Userspace interface could remain 64-bit? For example the size above is still 64 bit and all other ioctls like user_ptr pwrite, etc. expect 64 bit physical address type as well. 32-bit application should still use 64-bit data types for the ioctls.

Reply:
2.  if userspace interface remains 64-bit, it give build error 
 zocl_bo.c:651:36: error: passing argument 3 of 'zocl_bo_describe' from incompatible pointer type [-Werror=incompatible-pointer-types]
|   zocl_bo_describe(bo, &args->size, &args->paddr);

where args->paddr is uint64_t

Reply:
Change the interface of user space and kernel space is danger. We should only fix this issue in kernel space.

Reply:
Let's keep the interface unchanged. I am testing a new proposal, will send you update today.